histoDB/histodb-dataserver/pom.xml

```diff
@@ -137,6 +137,7 @@
         <dependency>
             <groupId>eu.itesla_project</groupId>
             <artifactId>cim1-import</artifactId>
+            <version>0.1-SNAPSHOT</version>
```

Comment:
any reason why the other references to hard-coded iidm-*  versions, in this very same pom.xml were not fixed during the idm-xxxx format merge?